draft-ietf-quic-transport.md

```diff
@@ -4008,8 +3992,8 @@ off-path injection as specified in {{!RFC8201}} and Section 5.2 of {{!RFC8085}}.
 This validation SHOULD use the quoted packet supplied in the payload of an ICMP
 message to associate the message with a corresponding transport connection (see
 Section 4.6.1 of {{!DPLPMTUD}}).  ICMP message validation MUST include matching
-IP addresses and UDP ports {{!RFC8085}} and, when possible, connection IDs to an
-active QUIC session.  The endpoint SHOULD ignore all ICMP messages that fail
+IP addresses and UDP ports ({{!RFC8085}}) and, when possible, connection IDs to
```

Comment:
Unlike above, this would seem to require the parenthetical.

Reply:
I think that the prevailing use of external references - those that appear in brackets - is both inline "defined in [REF]" and not as an automatic parenthetical, but an annotation of a noun in cases like "such as UDP [REF]", which you would convert to something like `<a href="REF">UDP</a>` were this text.  That's harder to codify though.  In many ways this is a limitation of markdown, but I don't see it as a serious shortcoming, because fixes (such as those in HTML or XML) just transfer the problem to the presentation layer.

Reply:
Ah, while I'm looking at the text output and saying "'UDP [UDP]' looks a bit strange; let's fix that."